chef/cookbooks/ohai/files/default/plugins/linux/s390x.rb

```diff
@@ -0,0 +1,40 @@
+provides "s390x"
+
+if File.exist? "/proc/sysinfo"
```

Comment:
I have not been able to find that file on s390x, armv7, aarch64 or ppc64le. so I guess it is unique enough. the content of the file only makes sense on s390x at least.